plugins/org.locationtech.udig.render.feature.basic/src/org/locationtech/udig/render/internal/feature/basic/BasicFeatureRenderer.java

```diff
@@ -430,9 +430,17 @@ private void render(Graphics2D graphics, ReferencedEnvelope bounds, IProgressMon
             rendererHints.put(StreamingRenderer.LABEL_CACHE_KEY,
                     new LabelCacheDecorator(labelPainter, origin, layerId));
 
+            if (RendererUtils.isAdvancedProjectionSupportEnabled()) {
+                rendererHints.put(StreamingRenderer.ADVANCED_PROJECTION_HANDLING_KEY, true);
```

Comment:
These option is for feature renderings such as shapefiles. What about raster datasets and WMS layers, any chance to allow enabling advanced projection support for these as well?